metrics/web/test/project_groups/presentation/widgets/add_project_group_card_test.dart

```diff
@@ -137,6 +143,36 @@ void main() {
       },
     );
 
+    testWidgets(
+      "applies the barrier color from the metrics theme to the backdrop color of the add project group dialog",
+      (WidgetTester tester) async {
+        final notifierMock = ProjectGroupsNotifierMock();
+        when(notifierMock.hasConfiguredProjects).thenReturn(true);
+        when(notifierMock.projectCheckboxViewModels).thenReturn([]);
+        when(notifierMock.projectGroupDialogViewModel).thenReturn(
+          ProjectGroupDialogViewModel(
+            selectedProjectIds: UnmodifiableListView([]),
+          ),
+        );
+
+        await mockNetworkImagesFor(
+          () => tester.pumpWidget(_AddProjectGroupCardTestbed(
+            projectGroupsNotifier: notifierMock,
+            theme: metricsTheme,
+          )),
+        );
+
+        await tester.tap(find.byType(AddProjectGroupCard));
+        await tester.pumpAndSettle();
+
+        final modalBarrier = tester.widget<ModalBarrier>(
+          find.byType(ModalBarrier).last,
+        );
+
+        expect(modalBarrier.color, barrierColor);
```

Comment:
Would it be more reliable? 
```suggestion
        final barrierFinder = find.byWidgetPredicate(
          (widget) => widget is ModalBarrier && widget.color == barrierColor,
        );

        expect(barrierFinder, findsOneWidget);
```